api/bases/test.openstack.org_tempests.yaml

```diff
@@ -373,6 +436,74 @@ spec:
                         of tempest. Please refer to Please refer to https://docs.openstack.org/tempest/latest/
                         for the further explanation of the CLI parameters.
                       properties:
+                        ExtraImageType:
```

Comment:
(if the changes above are implemented, this should be ExtraImagesType)

Reply:
good point, thanks!:)